rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -34,6 +34,13 @@
 
 MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent)
 {
+	QFile file("stylesheet.qss");
```

Comment:
Can you move the style sheet loading to the application class?  People that change fonts (cough talkashie) would probably want to have their game window have the style sheet modified too.